src/kirby/components/icon/icon.component.scss

```diff
@@ -5,6 +5,11 @@
   width: 1em;
   height: 1em;
   font-size: size('m');
+
+  svg path {
+    fill: currentColor;
+    stroke: currentColor;
+  }
 }
 
 :host(.sm) {
```

Comment:
how about when using icons in a badge ? should we not have a size for badge, the example on master the icon do not fit the badge on native and we might also need a smaller icon for "normal" size badge, so we have the size that are shown on the cookbook and one for the size of the texts examples, is that considered here ?

Reply:
@bdyken It's generally considered bad practice to comment on code/features outside the PR that the requester is asking for a review on. You should create  a GH issue (bug or feature request) instead. A PR/review is about the changes made, not the overall component/feature in question.